components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.device-organization/device-organization.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please change the license header's year.

Reply:
Fixed a6f6ad8032d41cd330a85e066ea029ed97f099ff